### PR TITLE
stripped out game.remove/removAll

### PR DESCRIPTION
**Pull request by agmcleod**
#327

Removed the me.game.remove() method. Updated documentation to use `me.game.world.removeChild`. 

@obiot wanted you to check it out first, given we should be careful on the remove function since the `me.game.remove` has the force option. Through my experience in using melon, the force is nothing really but trouble.

Testing against the different examples and some of my own it seems to be fine. The main thing is users will need to update their code to use removeChild.

### CODE REVIEW

File: tests/spec/droptarget.js [hunk 1]
```diff
@@ -59,7 +59,6 @@
                 // create a new droptarget entity instance
                 droptarget = new Droptarget(100, 100, {width: 200, height: 200});
                 // add the test droptarget to the game
-                me.game.world.addChild(droptarget, 1);
```
Comment (parasyte) [inline tests/spec/droptarget.js:62]: Why was this line removed?

Comment (agmcleod) [inline tests/spec/droptarget.js:62]: @parasyte Just added back. I did changes to those files, and then merged with the latest 1.0.0-dev. I had merge conflicts with those files, looks like i deleted one line extra by mistake ><
